test/fixtures/templater/jinja_j_libraries/jinja.sql

```diff
@@ -1,3 +1,3 @@
 SELECT 56
-FROM {{ foo.schema }}.{{ foo.table("xyz") }}
+FROM {{ foo.schema }}.{{ table_proxy("xyz") }}
```

Comment:
Could we add a test rather than change an existing one? In effect, this seems like we're removing an existing test, which would not be good.

Reply:
I thought about that, but my reasoning was that both the usage of `{{ foo.schema }}` and `{{ foo.table("xyz") }}` is actually testing the same thing, so in a way one of the two is superfluous. One reference to the `foo` package should suffice in testing that the library is available for the template.

By altering one reference to test the macro case, we keep it concise but still complete. 

But, if you still don't agree with this reasoning, I am happy to add it as a separate test

Reply:
I'd prefer a second test so we have a test using dots. Thismis a common reason for defining macros in Python. Tests are cheap, and this test also serves as a working example.